ote/src/cljc/ote/db/transport_service.cljc

```diff
@@ -67,7 +67,8 @@
 
   ["external_interface_search_result" ::external-interface-search-result]
   ["transport_service_search_result" ::transport-service-search-result
-   ote.db.modification/modification-fields])
+   ote.db.modification/modification-fields]
+  ["service_company" ::service-company])
```

Comment:
Map modification columns to `ote.db.modification` ns

Reply:
Do I have to?

Reply:
Somebody has to do it. We don't want multiple definitions for the same columns, it is confusing for future maintenance when sometimes the same column is in one namespace and other times in another.

Reply:
done